packages/sync-core/src/lib/TLSocketRoom.ts

```diff
@@ -311,12 +311,45 @@ export class TLSocketRoom<R extends UnknownRecord = UnknownRecord, SessionMeta =
 			},
 			log: this.log,
 		})
-
 		// replace room with new one and kick out all the clients
 		this.room = newRoom
 		oldRoom.close()
 	}
 
+	async associateFileAssets(fileId: string, bucket: any) {
```

Comment:
let's make this `@internal` for now.

honestly it should probably live in the room durable object but it's fine to keep it here as long as it's private.

Reply:
Pulled it to `TLDrawDurableObject`.